main.css

```diff
@@ -4,6 +4,36 @@
     box-sizing: border-box;
 }
 
+/* Mobile Styles */
+@media only screen and (max-width: 400px) {
+    body {
+        font-family: Arial;
```

Comment:
🔧  The `font-family` should probably not be declared within the media query. If you add it as a declaration on line 6, you won't have to repeat yourself. 

The reason why we'd want the `font-family` outside of the media queries is because we don't want someone to re-size their screen and then accidentally get a new font when it isn't expected.